src/swagger/auth/auth.docs.js

```diff
@@ -0,0 +1,106 @@
+/**
```

Comment:
The file extension should be `.swagger.js`. Also, move it out of the `auth` folder

Reply:
Ensure the file extension is reflected in the swagger config file

Reply:
Ok